### PR TITLE
ci: CI for shared bridge

### PR DESCRIPTION
**Pull request by ly0va**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.
- [ ] Linkcheck has been run via `zk linkcheck`.

### CODE REVIEW

File: .github/workflows/ci-core-reusable.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ jobs:
           ci_run zk contract build
 
       - name: Contracts unit tests
-        run: ci_run zk test l1-contracts
+        run: ci_run yarn l1-contracts test
```
Comment (mm-zk) [inline .github/workflows/ci-core-reusable.yml:62]: let's then remove the 'l1-contracts' from zk test..

(infrastructure/tests.ts)

File: core/lib/basic_types/src/protocol_version.rs [hunk 2]
```diff
@@ -45,8 +45,8 @@ pub enum ProtocolVersionId {
     Version21,
     Version22,
     Version23,
-    Version24,
     Local,
+    Version24,
```
Comment (mm-zk) [inline core/lib/basic_types/src/protocol_version.rs:49]: is this on purpose?

File: etc/env/configs/dev_validium_docker.toml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+[chain.state_keeper]
+compute_overhead_part = 1.0
+pubdata_overhead_part = 0.0
+batch_overhead_l1_gas = 1000000
+# This value should be higher for Validium, but now it is not compatible with the current blobs model.
+# For this config to be set to its proper value we need to fully integrate Validium + Blobs.
+max_pubdata_per_batch = 100000
+fee_model_version = "V2"
+l1_batch_commit_data_generator_mode = "Validium"
+miniblock_iteration_interval = 50
```
Comment (mm-zk) [inline etc/env/configs/dev_validium_docker.toml:20]: why is this here for docker?

**Review by mm-zk — APPROVED** [review]

File: core/lib/basic_types/src/protocol_version.rs (hunk 2 above)
Comment (ly0va) [reply core/lib/basic_types/src/protocol_version.rs:49]: yes! otherwise upgrade unit tests fail. There was a comment about it sometime ago, but apparently it was removed during merge
